src/ImuSensor_TEST.cc

```diff
@@ -436,6 +436,8 @@ TEST(ImuSensor_TEST, OrientationReference)
   // Create an ImuSensor
   auto sensor = mgr.CreateSensor<ignition::sensors::ImuSensor>(
       imuSDF);
+  sensor->SetWorldFrameOrientation(math::Quaterniond(0, 0, 0),
+    sensors::WorldFrameEnumType::ENU);
```

Comment:
why is this necessary?

Reply:
Currently, ``SetWorldFrameOrientation`` is also responsible for applying the "CUSTOM" tag. I could separate this logic in a separate method though.